docs/concepts/workloads/pods/disruptions.md

```diff
@@ -0,0 +1,260 @@
+are not limited by PDBs whe doing rolling upgrades -- the handling of failures
+during application updates is configured in the controller spec.
+(Learn about [updating a deployment](docs/concepts/cluster-administration/manage-deployment/#updating-your-application-without-a-service-outage).)
+
+When a pod is evicted using the eviction API, it is gracefully terminated (see
+`terminationGracePeriodSeconds` in [PodSpec](/docs/resources-reference/v1.6/#podspec-v1-core).)
+
+## PDB Example
+
+Consider a cluster with 3 nodes, `node-1` through `node-3`.
+The cluster is running several applications.  One of them has 3 replicas initially called
+`pod-a`, `pod-b`, and `pod-c`.  Another, unrelated pod without a PDB, called `pod-x`, is also shown.
+Initially, the pods are layed out as follows:
+
+| pod   | pod status | node   |  node status |
+|:-----:|:----------:|:------:|:------------:|
+| pod-a |  available | node-1 | normal       |
+| pod-x |  available | node-1 | normal       |
+| pod-b |  available | node-2 | normal       |
+| pod-c |  available | node-3 | normal       |
+
+All 3 pods are part of an deployment, and they collectively have a PDB which requires
+that there be at least 2 of the 3 pods available at all times.
+
+The cluster administrator want to reboot into a new kernel version to fix a bug in the kernel.
+The cluster administrator first tries to drain `node-1` using the `kubectl drain` command.
+That tool tries to evict `pod-a` and `pod-x`.  This succeeds immediately.
+Both pods go into the `terminating` state at the same time.
+This puts the cluster in this state:
+
+| pod   | pod status  | node   |  node status |
+|:-----:|:-----------:|:------:|:------------:|
+| pod-a | terminating | node-1 | draining     |
+| pod-x | terminating | node-1 | draining     |
+| pod-b |  available  | node-2 | normal       |
+| pod-c |  available  | node-3 | normal       |
+
+The deployment notices that there is a missing pod, so it creates a replacement,
+called `pod-d`.  Since `node-1` is cordoned, it lands on another node.
+Also, `pod-x` terminates, and we ignore its replacement.
+
+Now the cluster is in this state:
+
+| pod name |  scheduled where |  status   |
+|:--------:|:----------------:|:---------:|
+| pod-a    | node-1           | terminating |
+| pod-b    | node-2           | available |
+| pod-d    | node-2           | not available (starting up) |
+| pod-c    | node-3           | available |
+
+| pod   | pod status  | node   |  node status |
+|:-----:|:-----------:|:------:|:------------:|
+| pod-a | terminating | node-1 | draining     |
+| pod-b |  available  | node-2 | normal       |
+| pod-c |  available  | node-3 | normal       |
+| pod-d | not-available | node-2 | normal     |
+
+At this point, if an impatient cluster administrator tries to drain `node-2` or
+`node-3`, they will not succeed, because there are only 2 available pods for the deployment, and its PDB requires at least 2.
```

Comment:
Withdrawn! According to the the developer's style guide the singular they is correct.